packages/webapp/src/pages/Logs/ShowMessage.tsx

```diff
@@ -66,7 +66,7 @@ export const ShowMessage: React.FC<{ message: MessageRow }> = ({ message }) => {
                                 return { code: { padding: '0', whiteSpace: 'pre-wrap' } };
                             }}
                         >
-                            {JSON.stringify({ error: message.error || undefined, output: message.meta || undefined }, null, 2)}
+                            {JSON.stringify({ error: message.error?.message || undefined, output: message.meta || undefined }, null, 2)}
```

Comment:
is the `|| undefined` necessary?

Reply:
yes because messge.error can be `null` and null is not automatically removed from json stringify, but `undefined` will be stripped.